ide/extbrowser/src/org/netbeans/modules/extbrowser/NbDdeBrowserImpl.java

```diff
@@ -413,7 +414,7 @@ private String getFileUrl(String url) {
             while ((f == null) && (retries > 0)) {
                 retries--;
                 try {
-                    f = File.createTempFile("extbrowser", ".html");             // NOI18N
+                    f = Files.createTempFile("extbrowser", ".html").toFile();             // NOI18N
```

Comment:
this is usually more secure since it retains the file permissions of the process, however I just realized that this is windows so this doesn't matter...